test/test.js

```diff
@@ -1771,3 +1771,98 @@ test( "Validation triggered on radio and checkbox via click", function() {
 	// test if there is no error anymore
 	ok( form.find( "input.error" ).length === 0, "Form valid" );
 });
+
+test("file accept - image wildcard", function() {
+
+	function dummy() {
+		return file;
+	}
+
+	// https://developer.mozilla.org/en-US/docs/Web/API/FileList
+	var
+		file = {
+		name: "test.png",
+		size: 500001,
+		type: "image/png"
+	},
+		fileList = {
+		0: file,
+		length: 1,
+		item: dummy
+	},
+		input = {},
+		$form = $("<form />"),
+		proxy,
+		validator;
+
+	// dummy DOM element
+	input.type = "file";
```

Comment:
You can define these above, its fine to reference a variable defined in the same multi declaration.